config/testimonials/developers.yml

```diff
@@ -0,0 +1,9 @@
+shared:
+  -
+    name: Dusan
+    testimonial: Switching jobs and overcoming the period between contracts was extremely painful. railsdevs.com is precisely what I needed. Now companies come to me instead of me chasing the work. I am incredibly grateful Joe did such a favor to the Rails community and built this platform.
```

Comment:
Moving the testimonials into these config files made it _very_ hard to do the mid-sentence bolding that the designer put together. I thought of giving it a second key, but then you run into the challenge of figuring out where to render it in the VC template. There's some inelegant solutions we could pursue, but I wanted to get your thoughts on if it is worth it before doing that.